src/main/java/uk/gov/companieshouse/api/accounts/model/entity/BalanceSheetEntity.java

```diff
@@ -16,4 +16,15 @@ public Integer getCalledUpShareCapitalNotPaid() {
     public void setCalledUpShareCapitalNotPaid(Integer calledUpShareCapitalNotPaid) {
         this.calledUpShareCapitalNotPaid = calledUpShareCapitalNotPaid;
     }
+
+    @Field("fixed_assets")
+    private FixedAssetsEntity fixedAssets;
```

Comment:
Can you put all the global variables together please. This makes them easier to identify.